### ISSUE TITLE
k-Nearest Neighbours (kNN)

### ISSUE THREAD

**Issue by dianzrong** [uoft-tmi/signature-forgery-detection#5]
### Purpose
It's important to try simpler models like k-Nearest Neighbours before more intricate models to ensure we don't overly complicate our solution. 
### Task
- [ ] Understand, experiment with, and implement the structure for the kNN model
- [ ] Update the model to take in the processed images and return whether the signature is a forgery
- [ ] Train the model on the train dataset
- [ ] Fine-tune the model (test values of k, measures of distance, etc.) on the validation dataset
- [ ] Report the model's accuracy on the test set

### Resources

- [Intro to kNN](https://www.w3schools.com/python/python_ml_knn.asp)
- [Process of kNN](https://www.datacamp.com/tutorial/k-nearest-neighbor-classification-scikit-learn)
- [Possible parameters to fine-tune](https://www.baeldung.com/cs/k-nearest-neighbors)
